query/src/main/scala/hmda/query/model/filing/Msa.scala

```diff
@@ -33,6 +16,40 @@ case class MsaWithName(
     homePurchase: Int,
     homeImprovement: Int,
     refinance: Int
+) {
+  def addName: MsaWithName = MsaWithName(
```

Comment:
The return type on this method is pretty clear, I don't think it needs to be specified in the declaration.